tests/infra/remote.py

```diff
@@ -371,7 +371,13 @@ def setup(self, **kwargs):
 
     def get_cmd(self):
         cmd = " ".join(self.cmd)
-        return f"cd {self.root} && {self.env.keys()} {cmd} 1> {self.out} 2> {self.err} 0< /dev/null"
+        env = " && ".join(
+            f'export {key}={value.replace(os.linesep, "")}'
```

Comment:
As discussed yesterday, this doesn't work without `export` does it?

Reply:
Correct